src/com/homework/ChatService.java

```diff
@@ -1,43 +1,88 @@
 package com.homework;
 
-import java.util.Scanner;
-import java.util.Date;
 import java.text.SimpleDateFormat;
+import java.util.*;
 
 import static java.lang.System.exit;
 
+
 public class ChatService {
-    Scanner scan = new Scanner(System.in);
-    public String sms;
-
-
-    public void chooseOption() {
-        while (true) {
-            System.out.println("Press Number for Action\n1. Send message\n2. Change current user\n3. Exit chat");
-            String option = scan.nextLine();
-            if (option.equals("1")) {
-                sendMessage();
-            } else if(option.equals("2")){
-                break;
-            } else if (option.equals("3")) {
-                exitChat();
-            } else {
-                System.out.println("Something went wrong. Please be careful and choose one of the options below.");
-            }
+    public static ArrayList<ChatMember> activeMemberList = new ArrayList<>();
+    public static String history = "";
+    public static Scanner scan = new Scanner(System.in);
+    public static String sms ;
+    public static int quantity;
+    public static String currentMember = "";
+
```

Comment:
Points 1 and 2 apply to all member variables above

Reply:
Sorry, I have one more question. 
Every member will have its own sms and history? Did I get it right?

Reply:
No, rather every ChatService will have its own history and sms.

Reply:
okay